### PR TITLE
added a plot

### PR DESCRIPTION
**Pull request by MacDonaldBL**
yay

### CODE REVIEW

File: my-saa-doc.Rmd [hunk 1]
```diff
@@ -1,28 +1,15 @@
 ---
-title: "untitled"
+title: "Very Nice work"
```
Comment (bean368) [inline my-saa-doc.Rmd:2]: Hmmmm what do you think about this title?

File: my-saa-doc.Rmd [hunk 2]
```diff
@@ -1,28 +1,15 @@
...
-```{r cars}
-summary(cars)
-```
 
-## Including Plots
+```{r my-flower-plot, fig.cap="the prettiest plot", fig.height=8, fig.width=7}
+library(ggplot2)
 
-You can also embed plots, for example:
+ggplot(iris, aes(Petal.Length, Petal.Width, colour = Species)) + geom_point()
```
Comment (bean368) [inline my-saa-doc.Rmd:11]: Should we break this plot up by species?

File: my-saa-doc.Rmd (hunk 1 above)
Comment (MacDonaldBL) [reply my-saa-doc.Rmd:2]: Very good

**Comment by MacDonaldBL** [comment]
VERY GOOD JOB, GILLIAN!

**Pull request merged**